parl/__init__.py

```diff
@@ -15,5 +15,4 @@
 generates new PARL python API
 """
 from parl.framework import *
-from parl.utils import *
-from parl.plutils import *
+from parl.remote import remote_class
```

Comment:
How about importing `RemoteManager` here if the `remote_class` has been imported here.

Reply:
Sounds good. done